### PR TITLE
Bullseye

### LINKED ISSUE #19: Install fails on Bullseye because of undefined HOME / GOCACHE variable for
Since Bullseye, install via CLI doesnt include a HOME variable anymore.

This is to make it easier to spot issues that would only be triggered on webadmin : app scripts should not assume that HOME is defined

https://ci-apps-bullseye.yunohost.org/ci/job/211

```
227257 INFO DEBUG - GOPATH=/opt/yunohost/photoview/libheif/examples PKG_CONFIG_PATH=/opt/yunohost/photoview/libheif:/opt/yunohost/photoview/libheif/libde265/dist/lib/pkgconfig/ CGO_CFLAGS="-I/opt/yunohost/photoview/libheif" CGO_LDFLAGS="-L/opt/yunohost/photoview/libheif/libheif/.libs" LD_LIBRARY_PATH=/opt/yunohost/photoview/libheif/libheif/.libs /usr/bin/go build -o heif-test-go heif-test.go
227257 INFO DEBUG - build cache is required, but could not be located: GOCACHE is not defined and neither $XDG_CACHE_HOME nor $HOME are defined
227257 INFO DEBUG - make[2]: *** [Makefile:1204: heif-test-go] Error 1
```

The appropriate fix would probably be to define a GOCACHE directory somewhere

### PR DESCRIPTION
**Pull request by yalh76**
## Solution

- *Fix #19*
- *Apply last example_ynh*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -56,7 +55,7 @@ ynh_app_setting_set --app=$app --key=mapbox_token --value=$mapbox_token
 #=================================================
 # STANDARD MODIFICATIONS
 #=================================================
-# FIND PORTS
+# FIND AND OPEN A PORT
```
Comment (Jules-Bertholet) [inline scripts/install:58]: We're not opening the port in the firewall, just using it internally

File: README_fr.md [hunk 2]
```diff
@@ -11,15 +11,21 @@ Si vous n'avez pas YunoHost, regardez [ici](https://yunohost.org/#/install) pour
 
 ## Vue d'ensemble
 
-Galerie photos simple et facile à utiliser, faite pour les photographes
+Photoview is a simple and user-friendly photo gallery that's made for photographers and aims to provide an easy and fast way to navigate directories, with thousands of high resolution photos.
```
Comment (Jules-Bertholet) [inline README_fr.md:14]: Pour un README en français, c'e n'est pas très français…

**Review by Jules-Bertholet — CHANGES_REQUESTED** [review]

File: scripts/install (hunk 1 above)
Comment (yalh76) [reply scripts/install:58]: yes, but it's the default title of this section...

**Review by Jules-Bertholet — APPROVED** [review]

**Review by Jules-Bertholet — APPROVED** [review]

**Comment by yalh76** [comment]
image

image

**Pull request merged**